project/src/com/google/daggerquery/dagger_query_textproto.bzl

```diff
@@ -25,7 +25,7 @@ def _dagger_query_textproto_impl(ctx):
     ctx.actions.run_shell(
         inputs = src_jars,
         outputs = [ctx.outputs.out],
-        command = "unzip -p {src_jar} binding_graph.textproto > {out}".format(
+        command = "unzip {src_jar} '*.textproto' -d .; zip -R {out} '*.textproto'".format(
```

Comment:
I think you don't need the `-R` (recursive) flag, as your `textproto` files are written to the top folder.

Reply:
Yes, it looks unnecessary. I've just removed it ✔️